spec/helpers/pages.js

```diff
@@ -201,7 +201,7 @@ function IngestSettings() {
         },
 
         timezoneLabel: element(by.id('timezone')),
-        timezoneDeleteBtn: $$('.icon-trash').get(0),
+        timezoneDeleteBtn: $$('.icon-pencil').get(0),
```

Comment:
it's probably not the only pencil icon on the page, you might need something more specific. or you can try with `$$('.icon-pencil').last()` but not sure it will do the trick